components/backends/sglang/slurm_jobs/scripts/gb200-fp8.sh

```
@@ -71,6 +71,8 @@ if [ "$mode" = "prefill" ]; then
     if [[ "${USE_INIT_LOCATIONS,,}" == "true" ]]; then command_suffix="--init-expert-location /configs/prefill_dsr1-0528_in1000out1000_num40000.json"; fi
 
     DYN_SKIP_SGLANG_LOG_FORMATTING=1 \
+    export TORCH_DISTRIBUTED_DEFAULT_TIMEOUT=1800 \
+    export SGL_DG_CACHE_DIR="/configs/dgcache/3p1dcache" \
```

Shellcheck (0.11.0)

[error] This applies export to the variable named export, which is probably not what you want. Use a separate command or the appropriate `declare` options instead.

(SC2316)